src/tools/tidy/src/unit_tests.rs

```diff
@@ -1,11 +1,13 @@
-//! Tidy check to ensure `#[test]` and `#[bench]` are not used directly inside `core`.
+//! Tidy check to ensure `#[test]` and `#[bench]` are not used directly inside
+//! `core` or `alloc`.
 //!
 //! `#![no_core]` libraries cannot be tested directly due to duplicating lang
-//! items. All tests and benchmarks must be written externally in `core/{tests,benches}`.
+//! items. All tests and benchmarks must be written externally in
+//! `{coretests,alloctests}/{tests,benches}`.
```

Comment:
Instead of referring to `#![no_core]` libraries maybe instead refer to core and alloc? alloc is not `#![no_core]`.